Makefile

```diff
@@ -48,6 +48,13 @@ development:
 
 development_aks:
 	$(eval include global_config/development.sh)
+	$(eval REGION=UK South)
```

Comment:
REGION will be the same for all environments. Its declaration could be moved to the top of the Makefile